scripts/test_hard_gt.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+#
+
+# Script for making "test_hard_gt.jsonl"
+## https://github.com/facebookresearch/mmbt/issues/6
+
+import argparse
+import copy
+import json
+import os
+
+def get_args(parser):
+    parser.add_argument("--task", type=str, default="mmimdb", choices=["mmimdb", "vsnli", "food101"])
+    parser.add_argument("--mmdb_path", type=str, default="/home/ubuntu/data/mmimdb/mmimdb")
+    parser.add_argument("--food_path", type=str, default="/home/ubuntu/script/food101")
+    parser.add_argument("--hard_gt_ids_path", type=str, default="hard_gt_ids.json")
+
+
+def get_hard_gt(args):
+    # load id list
+    with open(args.hard_gt_ids_path) as f:
+        hard_gt_ids = json.load(f)
+    # get id list for specified task
+    for hard_gt_dict in hard_gt_ids:
```

Comment:
What about just turning the JSON object into a dict `{'mmidmb': [], 'food101': []}` and you can just do `hard_ids = hard_gt_id_dict[args.task]`

Reply:
Much simpler, thanks!